lib/cdo/contact_rollups/v2/pardot.rb

```diff
@@ -99,6 +101,31 @@ def batch_create_remaining_prospects
     [submissions, errors]
   end
 
+  # Deletes a list of prospects from Pardot. For Pardot API documentation, see
+  # http://developer.pardot.com/kb/api-version-4/prospects/#using-prospects.
+  # @param [Array[Integer]] prospect_ids of the prospects to be deleted.
+  # @return [Array[Integer]] the set of prospect_ids that failed to be deleted.
+  def self.delete_prospects(prospect_ids)
+    failed_prospect_ids = []
+
+    return prospect_ids unless CDO.rack_env? :production
```

Comment:
why do we only delete in prod?

Reply:
Sorry for delay, we're finally getting back to finishing this...we basically just want to make sure we don't accidentally delete data from Pardot when testing in other environments (eg, if we run the full pipeline in other environments, we basically just want to skip this step).